apps/dialtone-documentation/docs/components/scrollbars.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Scrollbars
+description: A directive that adds customized overlay scrollbars to any scrollable section.
+status: beta
+thumb: true
+image: assets/images/components/scroller.png
+---
+
+## Scrollbars directive
+
+Allows to add overlay scrollbars that will look the same for every browser. The directive sets up the scrollbars from the library [OverlayScrollbars](https://kingsora.github.io/OverlayScrollbars/).
+
+<code-well-header>
+  <div class="d-hmx164 d-w30p d-bar8 d-ba d-bc-black-300" v-dt-scrollbars>
+    <div v-for="item in items" class="item">
+      {{ item}}
+    </div>
+  </div>
+</code-well-header>
+
+<code-example-tabs
+htmlCode='
+<div class="d-hmx164 d-w30p d-bar8 d-ba d-bc-black-300 custom-scrollbars" data-overlayscrollbars="host" data-overlayscrollbars-initialize="true">
+  <div class="os-size-observer">
+    <div class="os-size-observer-listener"></div>
+  </div>
+  <div data-overlayscrollbars-viewport="scrollbarHidden overflowXHidden overflowYScroll" tabindex="-1">
+    <div class="item">user 1</div>
+    ...
+  </div>
+  <div class="os-scrollbar os-scrollbar-horizontal os-theme-dark os-scrollbar-auto-hide os-scrollbar-handle-interactive os-scrollbar-cornerless os-scrollbar-unusable">
+    <div class="os-scrollbar-track">
+      <div class="os-scrollbar-handle"></div>
+    </div>
+  </div>
+  <div class="os-scrollbar os-scrollbar-vertical os-theme-dark os-scrollbar-auto-hide os-scrollbar-handle-interactive os-scrollbar-visible os-scrollbar-cornerless">
+    <div class="os-scrollbar-track">
+      <div class="os-scrollbar-handle"></div>
+    </div>
+  </div>
+</div>
+'
+vueCode='
+<div class="d-hmx164 d-w30p d-bar8 d-ba d-bc-black-300" v-dt-scrollbars>
+  <div v-for="item in items" class="item">
+    {{ item}}
+  </div>
+</div>
+'
+/>
+
+## Usage
+
+To add customized overlay scrollbars to a scrollable section, simply include the `v-dt-scrollbars` directive on the desired element.
+Add a `max-height` to the element of you want to set up a vertical scrollbar, or a `max-width` if you want to set up a horizontal scrollbar.
+
+## Characteristics
+
+* Has an overlay style
```

Comment:
Maybe an explanation of what an overlay style actually means (scrollbar appears over top of the content rather than the scrollbar taking up space within the container)